packages/core/src/test/shared/utilities/osUtils.test.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from 'assert'
+import { isNewOsSession } from '../../../shared/utilities/osUtils'
+import { fs } from '../../../shared'
+import { isWin } from '../../../shared/vscode/env'
+import { TestFolder } from '../../testUtil'
+
+describe('isNewOsSession', () => {
+    if (!isWin()) {
```

Comment:
I assume this isn't tested because we don't have a windows computer to test on lol